app/static/src/app/components/help/MarkdownPanel.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div class="markdown-panel" v-html="rendered"></div>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, onMounted } from "vue";
+import MarkdownItMathjax from "markdown-it-mathjax";
+import { useStore } from "vuex";
+import * as Token from "markdown-it/lib/token";
+import * as Renderer from "markdown-it/lib/renderer";
+import MarkdownIt from "./MarkdownItImport";
+
+interface MathJaxWindow {
+  MathJax: {
+    typeset: () => void
+  }
+}
+
+export default defineComponent({
+    name: "MarkdownPanel",
+    props: {
+        markdown: Array
+    },
+    setup(props) {
+        const store = useStore();
+
+        const appHelpUrl = computed(() => {
+            return `${store.state.baseUrl}/help`;
+        });
+
+        const isAbsoluteUrl = (s: string) => {
```

Comment:
my only real comment is that perhaps this bit (the mardown rewriting) could be pulled into a standard module so that if we want to reuse it later separate from the component that'll be easier. But if this component is going to be the one that we need for the generic help, no big reason to do that...

Reply:
Yeah, the plan is that the component itself should be reusable - we can pull out any bits from it for other things to use if we need to.